doc/scapy/usage.rst

```diff
@@ -520,7 +520,7 @@ Sniffing
 .. index::
    single: sniff()
 
-We can easily capture some packets or even clone tcpdump or tethereal. If no interface is given, sniffing will happen on every interfaces::
+We can easily capture some packets or even clone tcpdump or tethereal. Either one interface or a list of interfaces to sniff on can be provided. If no interface is given, sniffing will happen on every interface::
```

Comment:
Done the first, and also fixed a broken link to an image. Regarding the duplicate '::', I think they are needed to signal the code paragraph afterwards. 
From PEP 0287:

> Literal blocks (block-level literal text, such as code excerpts or ASCII graphics) are indented, and indicated with a double-colon ("::") at the end of the preceding paragraph (right here -->):